SampleApp/RoutingViewController.swift

```diff
@@ -48,41 +68,76 @@ class RoutingViewController: UIViewController, RoutingSearchDelegate {
+      self.routingLocale = Locale.init(identifier: "es_ES")
+    }))
+    actionSheet.addAction(UIAlertAction.init(title: "Czech", style: .default, handler: { [unowned self] (action) in
+      self.routingLocale = Locale.init(identifier: "cs-CZ")
+    }))
+    actionSheet.addAction(UIAlertAction.init(title: "Italian", style: .default, handler: { [unowned self] (action) in
+      self.routingLocale = Locale.init(identifier: "it_IT")
+    }))
+    actionSheet.addAction(UIAlertAction.init(title: "German", style: .default, handler: { [unowned self] (action) in
+      self.routingLocale = Locale.init(identifier: "de-DE")
+    }))
+    actionSheet.addAction(UIAlertAction.init(title: "Slovenian", style: .default, handler: { [unowned self] (action) in
+      self.routingLocale = Locale.init(identifier: "sl-SI")
+    }))
+    actionSheet.addAction(UIAlertAction.init(title: "Pirate", style: .default, handler: { [unowned self] (action) in
+      self.routingLocale = Locale.init(identifier: "pirate")
+    }))
+    actionSheet.addAction(UIAlertAction.init(title: "Cancel", style: .cancel, handler: { [unowned self] (action) in
+      self.dismiss(animated: true, completion: nil)
+    }))
+    self.navigationController?.present(actionSheet, animated: true, completion: nil)
+  }
 
-    let endingPoint = OTRRoutingPoint(coordinate: OTRGeoPointMake(point.coordinate.latitude, point.coordinate.longitude), type: .break)
+  private func requestRoute() {
+    guard let routingController = try? MapzenRoutingController.controller() else { return }
+    routingController.updateLocale(routingLocale)
 
-    routingController.requestRoute(withLocations: [startingPoint, endingPoint],
-                                                costingModel: .auto,
-                                                costingOption: nil,
-                                                directionsOptions: ["units" : "miles" as NSObject]) { (routingResult, token, error) in
-                                                  print("Error:\(error)")
-                                                  self.currentRouteResult = routingResult
-                                                  self.routeResultTable?.show(routingResult!)
+    guard let currentLocation = LocationManager.sharedManager.currentLocation, let destination = destination else { return }
+    let startingPoint = OTRRoutingPoint(coordinate: OTRGeoPointMake(currentLocation.coordinate.latitude, currentLocation.coordinate.longitude), type: .break)
+    let endingPoint = OTRRoutingPoint(coordinate: destination, type: .break)
+
+    _ = routingController.requestRoute(withLocations: [startingPoint, endingPoint],
+                                       costingModel: .auto,
+                                       costingOption: nil,
+                                       directionsOptions: ["units" : "miles" as NSObject]) { (routingResult, token, error) in
```

Comment:
Yes, the next step will be adding support for a units enum: https://github.com/mapzen/on-the-road_ios/issues/27